src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/global.scss

```diff
@@ -0,0 +1,20 @@
+@import 'normalize.css/normalize.css';
+@import './colors.scss';
+
+$bodyBackgroundColor: $wildSand;
+$h1TextColor: $blueZodiac;
+
+*, 
+*:after,
+*:before { 
```

Comment:
[Tests](https://travis-ci.org/sulu/sulu/jobs/259733727#L332) are already failing because of that 😉